locales/locales.php

```diff
@@ -961,15 +961,15 @@ public function __construct() {
 		$gu->google_code = 'gu';
 		$gu->facebook_locale = 'gu_IN';
 
-		$ha = new GP_Locale();
-		$ha->english_name = 'Hausa';
-		$ha->native_name = 'هَوُسَ';
-		$ha->lang_code_iso_639_1 = 'ha';
-		$ha->lang_code_iso_639_2 = 'hau';
-		$ha->slug = 'ha';
-		$ha->text_direction = 'rtl';
-		$ha->google_code = 'ha';
-		$ha->facebook_locale = 'ha_NG';
+		$hau = new GP_Locale();
+		$hau->english_name = 'Hausa';
+		$hau->native_name = 'Harshen Hausa';
```

Comment:
Languages are evolving. The locale was originally added as an Arabic script, including text direction. Both script and text direction are changed in this version at the request of the person who will maintain the locale. The native name is not in English, it's in boko, which is a latin alphabeth used for Hausa. https://en.wikipedia.org/wiki/Boko_alphabet